go.mod

```diff
@@ -63,26 +49,21 @@ require (
 	k8s.io/apiextensions-apiserver v0.0.0-20190626210203-fdc73e13f9a6
 	k8s.io/apimachinery v0.0.0
 	k8s.io/apiserver v0.0.0-20190625052034-8c075cba2f8c
-	k8s.io/cli-runtime v0.0.0-20190314001948-2899ed30580f
-	k8s.io/client-go v11.0.0+incompatible
-	k8s.io/cloud-provider v0.0.0-20190323031113-9c9d72d1bf90 // indirect
-	k8s.io/code-generator v0.0.0
+	k8s.io/cli-runtime v0.0.0-20190711111425-61e036b70227
+	k8s.io/client-go v12.0.0+incompatible
```

Comment:
How was this version derived? It fails [validation in Go 1.13](https://tip.golang.org/doc/go1.13#version-validation): the `+incompatible` annotation is only supported for versions that lack a `go.mod` file, and the version of `client-go` tagged `v12.0.0` does include such a file.

(See https://github.com/kubernetes/client-go/issues/631 and https://github.com/golang/go/issues/33558.)

Reply:
@willb we are aware of this issue and will be fixed in the next update of the dependencies

> How was this version derived?

From the examples in client go. If you check the required section we have a valid commit https://github.com/kubernetes/ingress-nginx/blob/master/go.mod#L72

> It fails validation in Go 1.13:

Sure, but that is a change that a not released version go is introducing. Before 1.12.6 this was ok.